subprojects/docs/src/docs/release/notes.md

```diff
@@ -48,6 +48,10 @@ In this release, Gradle's incremental C/C++ compilation is now able to understan
 
 When registering task properties via the runtime API, property names are not required to be Java identifiers anymore, and can be any non-empty string.
 
+### ANTLR task is now cacheable by default
+
+When generating grammar sources with ANTLR, now the task's outputs are stored and retreived from the build cache.
```

Comment:
misspelling, `retrieved`

Reply:
Should we also add this task to the list of cacheable tasks in the build cache chapter?

Reply:
Good point, added, fixed.